.gitignore

```diff
@@ -17,7 +17,8 @@ CMakeCache.txt
 CMakeFiles
 CMakeScripts
 Testing
-Makefile
+/Makefile
+/build/Makefile
```

Comment:
I needed to push 2 Makefiles in the `docker` and `remote` directory so we can not ignore `Makefile` everywhere.